libs/python/setup.py.in

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
```

Comment:
is this file supposed to be here? i'm not really sure what a .py.in filetype is. this looks like an ordinary python file

Reply:
i think this file is auto generated by pyconfigure?